tests/manual-test-cases/Group5-Functional-Tests/5-25-OPS-User-Grant.robot

```diff
@@ -107,11 +122,7 @@ granted ops-user perms work after upgrade
     Check Upgraded Version
 
     # Run a govc test to check that access is denied on some resources
-    Log To Console  Running govc to set drs-enabled, it should fail
-    ${rc}  ${output}=  Run And Return Rc And Output  GOVC_USERNAME=${ops_user_name} GOVC_PASSWORD=${ops_user_password} govc cluster.change -drs-enabled /${datacenter}/host/${cluster}
-    Log  Govc output: ${output}
-    Should Be Equal As Integers  ${rc}  1
-    Should Contain  ${output}  Permission to perform this operation was denied
+    Attempt To Create Resource Pool
```

Comment:
I think this can be `Attempt To Disable DRS`, can't it?

Reply:
Could be, but it was basically a merge of your additions and @anchal-agrawal's additions, so I left both `Attempt To Create Resource Pool` and `Attempt To Disable DRS` in there for variety. Still trying to get `granted ops-user perms work after upgrade` to pass, so I'll clean it up as I go forward.

Reply:
Ah, disabling DRS requires less privilege than creating a resource pool. Usually disabling DRS makes a good sanity check that we're not granting the operations user privileges they don't need.

In the specific case that we're using `--affinity-vm-group`, we currently have to grant the operations user the privilege that lets them disable DRS (as that's the same privilege that lets them manage affinity constructs), so we check creating a resource pool instead.